.dockerignore

```diff
@@ -0,0 +1,4 @@
+Dockerfile
+.dockerignore
+build/
```

Comment:
Please add at least CLion files - maybe use gitignore.io?

Reply:
I can include the whole .idea folder, but none of this copies over to the final (alpine) image anyway - and I can't think of a big reason to only build the intermediary (ubuntu) image

Reply:
I have now excluded everything by default except the lib folder and main.cc/cmakelists (i.e. all build necessary kit)